sonar-xml-plugin/src/main/java/org/sonar/plugins/xml/checks/XmlSchemaCheck.java

```diff
@@ -257,7 +259,8 @@ private void validate(String[] schemaList) {
       }
 
     } catch (IOException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1166)